src/js/post-911-gib-status/reducers/index.js

```diff
@@ -29,6 +32,8 @@ function post911GIBStatus(state = initialState, action) {
       return set('availability', 'noChapter33Record', state);
     case GET_ENROLLMENT_DATA_FAILURE:
       return set('availability', 'unavailable', state);
+    case SET_SERVICE_UP:
```

Comment:
This might be misleading since we could call `SET_SERVICE_UP` with a `down` status. Options:
1. Change the name to `SET_SERVICE_UP_STATUS` or something describing that this is the availability of the service
    - `availability` is already taken, which makes this a bit awkward for naming
2. Make multiple actions
    - `SET_SERVICE_PENDING` (awkward name)
    - `SET_SERVICE_UP`
    - `SET_SERVICE_DOWN`

Reply:
I'd be in favor of number 2 but with a slight rename.  Maybe:

- `SERVICE_STATUS_PENDING`
- `SERVICE_STATUS_UP`
- `SERVICE_STATUS_DOWN`